ec/acc/deployment_basic_defaults_test.go

```diff
@@ -165,6 +167,24 @@ func TestAccDeployment_basic_defaults(t *testing.T) {
 					resource.TestCheckResourceAttr(resName, "enterprise_search.#", "0"),
 				),
 			},
+			{
+				// Change the Elasticsearch resource deployment template to
+				// CCS, use defaults.
+				Config: ccsCfg,
```

Comment:
Actually this test will fail, because it's going from a Hot / Warm, where `node_attributes` are set, to using a CCS deployment_template which does not have `node_attributes`.

This test should be a standalone test and not a test which requires on a previous state.

Reply:
thanks @marclop will move this to a separate test case.

Reply:
have updated issue #129 with the error message

Reply:
Hi @marclop have created new test case file for it.